gammapy/datasets/make.py

```diff
@@ -83,32 +83,55 @@ def sigma_energy_theta(energy, theta, sigma):
 
 
 def make_test_observation_table(observatory_name='HESS', n_obs=10,
-                                datestart=None, dateend=None,
+                                az_range=Angle([0, 360], 'degree'),
+                                alt_range=Angle([45, 90], 'degree'),
+                                date_range=(Time('2010-01-01T00:00:00',
+                                                 format='isot', scale='utc'),
```

Comment:
Remove `format='isot', scale='utc'` ... UTC is the default and the format it's important.